File: cash/mario.py
from cs50 import get_int

def main():
    n = get_positive_int()
    pattern(n)

def get_positive_int():
    while(True):
        m = get_int("height:")
        if m > 0 and m < 9:
            break
    return m

def pattern(n):
    for i in range(n):
        for j in range(n):
            if i+j >= n-1:
                print("#",end="")
            else:
                print(" ",end="")
        print()

main()

File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
   int n;
   do
   {
       n=get_int("size:");
       if (n==9)
      { n=2;
      }
   }
   while(n<1);
   for(int i=0;i<n;i++)
   {
       for(int j=0;j<n;j++)
       {
           if(i+j>=n-1)
           {
               printf("#");
           }
           else
           {
               printf(" ");
           }
       }
       printf("\n");
   }
}
